recipes/ckanweb-deploy-exts.rb

```diff
@@ -108,6 +108,16 @@
 	package p
 end
 
+bash "Install NPM and NodeJS" do
+	code <<-EOS
+		if ! (yum install -y npm); then
+			# failed to install from standard repo, try a manual setup
+			curl --silent --location https://rpm.nodesource.com/setup_10.x | bash -
```

Comment:
is ckan really stuck on node v10?

Reply:
I think it's just used to generate some files, not to actually serve anything.

Reply:
please make this surfaced as a configurable variable with default to 10 so its future proofed.